tfjs-backend-webgpu/src/backend_webgpu.ts

```diff
@@ -856,6 +891,41 @@ export class WebGPUBackend extends KernelBackend {
     passEncoder.endPass();
   }
 
+  recordFromPixelsImportCommands(
```

Comment:
Can it be merged with `recordFromPixelsCommands` like below?
```
  recordFromPixelsCommands(output: GPUBuffer, program: FromPixelsProgram) {
    const bindGroup = this.device.createBindGroup({
      layout: program.bindGroupLayout,
      entries: [
        {
          binding: 0,
          resource: {
            buffer: output,
          }
        },
        {
          binding: 1,
          resource: program.GetInputTexture(),
        },
        {
          binding: 2,
          resource: {
            buffer: program.uniform,
          }
        }
      ],
    });
    this.ensureCommandEncoderReady();
    const passEncoder = this.currentCommandEncoder.beginComputePass();
    passEncoder.setPipeline(program.pipeline);
    passEncoder.setBindGroup(0, bindGroup);
    passEncoder.dispatch(
        program.dispatch[0], program.dispatch[1],
        program.dispatch[2]);
    passEncoder.endPass();
  }
```
And you can have different implementations for `program.GetInputTexture()`.

Reply:
That would be great!